mesonbuild/modules/hotdoc.py

```diff
@@ -155,6 +155,18 @@ def process_extra_extension_paths(self):
     def replace_dirs_in_string(self, string):
         return string.replace("@SOURCE_ROOT@", self.sourcedir).replace("@BUILD_ROOT@", self.builddir)
 
+    def process_gi_c_source_roots(self):
+        if self.hotdoc.run_hotdoc(['--has-extension=gi-extension']) != 0:
+            return
+
+        value, _ = self.get_value([list, str], 'gi_c_source_roots', default=[], force_list=True)
+        value.extend([
+                os.path.join(self.state.environment.get_source_dir(), self.interpreter.subproject_dir, self.state.subproject),
```

Comment:
__[Flake8]__ [E126] continuation line over-indented for hanging indent ([link](https://sider.review/gh/repos/19784232/pulls/5023#issue-3810684))

<sub>posted by Sider</sub>